frontend/src/components/pages/ProductCardPage.tsx

```diff
@@ -78,14 +66,14 @@ const ProductCardPage = () => {
         <Grid container spacing={3}>
           <Grid size={{ xs: 12, md: 6 }}>
             <Paper elevation={10}>
-              <ImageGallery images={mockData.images} />
+              <ImageGallery images={product.photos} />
             </Paper>
           </Grid>
 
           <Grid size={{ xs: 12, md: 6 }}>
             <Typography variant="h5" gutterBottom>{product.title}</Typography>
             <Typography variant="body1" component="p" >{product.description}</Typography>
-            <Typography variant="body2" gutterBottom><strong>Категория:</strong> {product.category}</Typography>
+            <Typography variant="body2" gutterBottom><strong>Категория:</strong> {categoryToRussian(product.category)}</Typography>
             <Typography variant="body2" gutterBottom><strong>Цена:</strong> {product.price}</Typography>
```

Comment:
Добавь рубль до кучи:
```suggestion
            <Typography variant="body2" gutterBottom><strong>Цена:</strong> {product.price} ₽</Typography>
```